usaspending_api/api_contracts/contracts/search/SpendingByTransactionCount.md

```diff
@@ -0,0 +1,41 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# IDV Funding [/api/v2/search/spending_by_transaction_count/]
+
+Returns the counts of transaction records which match the keyword grouped by award categories.
+
+## POST
+
++ Request (application/json)
+    + Attributes (object)
+        + `filters` (required, FilterObject)
+            Need to provide `keywords`
+
++ Response 200 (application/json)
+    + Attributes
+        + results (required, object)
+            - idvs: 1 (required, number)
+            - loans: 100 (required, number)
+            - direct_payments: 80 (required, number)
+            - contracts: 9 (required, number)
+            - other: 0 (required, number)
+            - grants: 0 (required, number)
+
+    + Body
+
+            {
+                "results": {
+                    "idvs": 597,
+                    "loans": 1,
+                    "direct_payments": 0,
+                    "contracts": 100732,
+                    "other": 0,
+                    "grants": 35
+                }
+            }
+
+# Data Structures
+
+## FilterObject (object)
++ `keywords`: `lockheed` (required, array[string], fixed-type)
```

Comment:
This endpoint accepts only a single keyword. (Same for the other keyword search endpoints)

Reply:
Actually..... it accepts both a string or an array.
`keyword` is a string
`keywords` is a string array

`keyword` is for backward compatibility

Reply:
Gotcha! When I mock the API and perform a keyword search on the frontend it returns a 404. Could we include both and make them optional?

Reply:
Since `keywords` passes the compliance test and is the most recent implementation, I'm good to leave it as-is.